Ibidapo/send-it
# #162442655 Implement Fetch API to Sign in & Register user on the application

#### What does this PR do?
- Implements the Fetch  API to 

#### How should this be manually tested?
- Visit the front-end of the application at https://ibidapo.github.io/send-it/UI/
- If a new user, click on **Sign up Tab** and enter valid credentials
- or as an existing user, click on **Sign in Tab** and enter valid credentials
- you should be redirected to the user page.

#### What are the relevant pivotal tracker stories?
#25
